src/coreclr/tools/superpmi/superpmi-shared/methodcontext.cpp

```diff
@@ -3558,70 +3581,13 @@ void MethodContext::recGetFieldAddress(CORINFO_FIELD_HANDLE field, void** ppIndi
         value.ppIndirection = CastPointer(*ppIndirection);
     value.fieldAddress      = CastPointer(result);
 
-    value.fieldValue = (DWORD)-1;
```

Comment:
I think all the code around `isReadyToRunCompilation` is dead now, so you can remove that too.

Reply:
(Having to even check that here was kind of a good indication that the JIT-EE interface was poorly designed in this area)